File: app/Http/Controllers/producer.php
<?php
require_once 'C:\xampp\htdocs\login\vendor\autoload.php';
use PhpAmqpLib\Connection\AMQPStreamConnection;
use PhpAmqpLib\Message\AMQPMessage;

$connection = new AMQPStreamConnection('localhost', 5672, 'guest', 'guest');
$channel = $connection->channel();

$msg = new AMQPMessage($argv);
$array = $argv[1];
// $cola = $array[7];
$channel->queue_declare($cola, false, false, false, false);
$channel->basic_publish($msg, '', $cola);
$channel->close();
$connection->close();

File: app/scrips/producer.py
#!/usr/bin/env python
# import pika
# import sys

# host_name = "localhost"
# message = sys.argv[1]
# # cola = sys.argv[2]
# cola = '1'

# connection = pika.BlockingConnection(
#     pika.ConnectionParameters(host=host_name))
# channel = connection.channel()

# channel.queue_declare(queue=cola, durable=False)

# channel.basic_publish(
#     exchange='',
#     routing_key=cola,
#     body=message,
#     properties=pika.BasicProperties(
#         delivery_mode=2,  # make message persistent
#     ))
# print(" [x] Sent %r" % message)
# connection.close()

import pika

connection = pika.BlockingConnection(
    pika.ConnectionParameters(host='localhost'))
channel = connection.channel()

channel.queue_declare(queue='hello')

channel.basic_publish(exchange='', routing_key='hello', body='Hello World!')
print(" [x] Sent 'Hello World!'")
connection.close()